pyproject.toml

```
@@ -1,2 +1,2 @@
 [build-system]
-requires = ["setuptools", "wheel", "numpy<1.22", "cython"]
+requires = ["setuptools", "wheel", "numpy<1.24", "cython"]
```

I was hoping to just get rid of the version requirement here
```suggestion
requires = ["setuptools", "wheel", "numpy", "cython"]
```